code/modules/mob/living/carbon/human/species/station/station_special_vr.dm

```diff
@@ -123,45 +123,45 @@
-						else //otherwise, just tell them to hide.
-							H << "<span class='info'> ...safe...</span>"
-				else // in light
+		if(light_amount <= 0.5) // in the darkness. No need for custom scene-protection checks as it's just an occational infomessage.
+			if(prob(2)) //periodic nagmessages just to remind 'em they're still feral
+				if (H.traumatic_shock >=min(60, H.nutrition/10)) // if hurt, tell 'em to heal up
+					H << "<span class='info'> This place seems safe, secure, hidden, a place to lick your wounds and recover...</span>"
+				else if(H.nutrition <= 100) //If hungry, nag them to go and find someone or something to eat.
+					H << "<span class='info'> Secure in your hiding place, your hunger still gnaws at you. You need to catch some food...</span>"
+				else if(H.jitteriness >= 100)
+					H << "<span class='info'> sneakysneakyyesyesyescleverhidingfindthingsyessssss</span>"
+				else //otherwise, just tell them to keep hiding.
+					H << "<span class='info'> ...safe...</span>"
+		else // must be in a lit area
+			var/list/nearby = oviewers(H)
+			if (nearby.len) // someone's nearby
+				if(prob(1)) // 1 in 100 chance of doing something so as not to interrupt scenes
+					var/mob/M = pick(nearby)
+					if (H.traumatic_shock >=min(60, H.nutrition/10)) //tell 'em to be wary of a random person
+						H << "<span class='danger'> You're hurt, in danger, exposed, and [M] looks to be a little too close for comfort...</span>"
+					else if(H.nutrition <= 250 || H.jitteriness > 0) //tell them a random person in view looks like food. It CAN happen when you're not hungry enough to be feral, especially if coffee is involved.
+						H << "<span class='danger'> Every movement, every flick, every sight and sound has your full attention, your hunting instincts on high alert... In fact, [M] looks extremely appetizing...</span>"
+					if(H.stat == CONSCIOUS)
+						H.emote("twitch")
 					if(!H.handling_hal)
 						spawn H.handle_feral()
-					if(prob(1)) //periodic nagmessages
-						if(H.nutrition <= 100) //If hungry, nag them to go and find someone or something to eat.
-							H << "<span class='danger'> Confusing sights and sounds and smells surround you - scary and disorienting it may be, but the drive to hunt, to feed, to survive, compels you.</span>"
-							if(H.stat == CONSCIOUS)
-								H.emote("twitch")
-						else if(H.jitteriness >= 100)
-							H << "<span class='danger'> yesyesyesyesyesyesgetthethingGETTHETHINGfindfoodsfindpreypounceyesyesyes</span>"
-							if(H.stat == CONSCIOUS)
-								H.emote("twitch")
-						else //otherwise, just tell them to hide.
-							H << "<span class='danger'> Confusing sights and sounds and smells surround you, this place is wrong, confusing, frightening. You need to hide, go to ground...</span>"
-							if(H.stat == CONSCIOUS)
-								H.emote("twitch")
+			else // nobody around
+				if(!H.handling_hal)
+					spawn H.handle_feral()
```

Comment:
Why spawn? Also this is nonstandard syntax for this.
```
spawn(0)
    H.handle_feral()
```
The way you did it might work but it's not how it's done everywhere else.

Reply:
Looks like it might be a depricated way of doing it. It's copypasted from the hallucination code. will change when I get home.

Reply:
You din answer tho, why spawn at all?

Reply:
Per discussion in chat, just remove the spawn since it's not necessary in your application.